indicators/management/commands/qa_program_widgets/qa_widgets.py

```diff
@@ -137,8 +137,15 @@ def __init__(self, program, country):
         self.sadd_disagg_labels = self.sadd_disagg_obj.disaggregationlabel_set.all()
 
     def create_standard_indicators(self, **kwargs):
-        indicator_ids = self.create_indicators(self.standard_params_base, **kwargs)
-        indicator_ids.extend(self.create_indicators(self.null_supplements_params, **kwargs))
+        try:
```

Comment:
Default arg to pop will save you a whole try catch here (`passed_apply_skips = kwargs.pop('apply_skips', None)`)

Reply:
Yeah it will